plugins/include/open62541/plugin/create_certificate.h

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2021 (c) Christian von Arnim, ISW University of Stuttgart (for VDW and umati)
+ */
+
+#ifndef CREATE_CERTIFICATE_H_
+#define CREATE_CERTIFICATE_H_
+
+#include <open62541/plugin/log.h>
+#include <open62541/types.h>
+
+#ifdef UA_ENABLE_ENCRYPTION
+enum UA_CertificateFormat {
+    UA_CERTIFICATE_FORMAT_DER,
+    UA_CERTIFICATE_FORMAT_PEM
+};
+
+/**
+ * (Experimental) Create a self-signed certificate
+ *
+ * It is recommended to store the generated certificate in a non volatile memory and
```

Comment:
volatile has a specific meaning in C.
If you mean "store on disk for reuse", just say so.